frontend/e2e-test/test/e2e/dev-api/types.ts

```diff
@@ -175,6 +177,10 @@ export interface CareArea {
   subCostCenter: string
 }
 
+interface OpenEndedPeriod {
```

Comment:
IMHO it would be better to use the same name as in the service code

Reply:
yes, but that is Period, which already exists here with a different meaning.. I suggest we fix this as part of the larger daterange refactoring

Reply:
OK, works for me